packages/zowe-explorer-ftp-extension/CHANGELOG.md

```diff
@@ -5,6 +5,14 @@ All notable changes to the "zowe-explorer-ftp-extension" extension will be docum
 ### New features and enhancements
 
 - Updated Zowe SDKs to `8.20.0` for technical currency. [#3526](https://github.com/zowe/zowe-explorer-vscode/pull/3526)
+
+### Bug fixes
+
+## `3.2.0`
+
+### New features and enhancements
+
+- Updated Zowe SDKs to `8.16.0` for technical currency. [#3526](https://github.com/zowe/zowe-explorer-vscode/pull/3526)
```

Comment:
Should this say 8.20.0, and can the duplicate line in the "TBD Release" section be removed?